applications/nrf5340_audio/src/modules/CMakeLists.txt

```diff
@@ -4,20 +4,16 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-set(SRCS
-  ${CMAKE_CURRENT_SOURCE_DIR}/audio_i2s.c
-  ${CMAKE_CURRENT_SOURCE_DIR}/audio_usb.c
-  ${CMAKE_CURRENT_SOURCE_DIR}/button_handler.c
-  ${CMAKE_CURRENT_SOURCE_DIR}/hw_codec.c
-  ${CMAKE_CURRENT_SOURCE_DIR}/led.c
-  ${CMAKE_CURRENT_SOURCE_DIR}/power_meas.c
-  ${CMAKE_CURRENT_SOURCE_DIR}/sd_card.c
-)
+target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/audio_i2s.c)
+target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/audio_usb.c)
+target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/button_handler.c)
+target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/led.c)
+target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/sd_card.c)
```

Comment:
Shouldn't the SD card be dependent on the nRF5340_audio_DK somehow?

Reply:
Good point - it should be possible to not compile `sd_card.c`, but I'm not sure if audio DK is the correct way to filter it though. If a kit has a different SD card slot the same code could be used, right?

Reply:
Added a new kconfig `NRF5340_AUDIO_SD_CARD_MODULE` that is set automatically for the audio DK via the board overlays